src/Microsoft.Health.Dicom.Core/DicomCoreResource.resx

```diff
@@ -627,4 +627,8 @@ For details on valid range queries, please refer to Search Matching section in C
   <data name="UnsupportedBuffering" xml:space="preserve">
     <value>Cannot buffer items if parallelism is unbounded.</value>
   </data>
+  <data name="InvalidTagValueWithFuzzyMatch" xml:space="preserve">
+    <value>Invalid query: specified tag '{0}' value '{1}' contains unsupported character '{2}'. Character is not supported with Fuzzy match, try exact match or removing the invalid character. </value>
```

Comment:
Do we want to include `value` in the message? Won't this be considered customer provided data?

Reply:
I am hoping the trace filter will filter it for logs we get.